src/components/settings/FontSizeOptions.tsx

```diff
@@ -14,6 +14,10 @@ const List = styled.ul`
   display: table;
   width: 100%;
   padding: 15px 0;
+  &:hover {
+    background-color: #F5F5F5;
```

Comment:
what are these colors? Can they be added to theme?

Reply:
I'll Clean up this PR